doc/man1/openssl-pkcs12.pod.in

```diff
@@ -345,6 +347,15 @@ then both, the private key and the certificates are encrypted using triple DES.
 
 Specify the MAC digest algorithm. If not included SHA256 will be used.
 
+=item B<-pbmac1_pbkdf2>
+
+Use PBMAC1 with PBKDF2 for MAC protection of the PKCS#12 file.
+
+=item B<-pbmac1_pbkdf2_md> I<digest>
+
+Specify the PBKDF2 KDF digest algorithm. If not included SHA256 will be used.
+Until C<-pbmac1_pbkdf2> is specified, this parameter is ignored.
```

Comment:
"Unless C<-pbmac1_pbkdf2> is specified,"

Reply:
Fixed, thanks